client/cmd/status_test.go

```diff
@@ -11,6 +11,18 @@ import (
 	"github.com/netbirdio/netbird/version"
 )
 
+var _ = initTZ()
+
+func initTZ() bool {
```

Comment:
It is not clear for me why do not you put this logic into a init() function instead of initTZ() and why return it back with unused bool.

Reply:
I've updated to use the init function